tests/components/shared/form/CheckboxComponent.test.js

```diff
@@ -0,0 +1,37 @@
+import { shallow } from 'enzyme';
+import CheckboxComponent from 'components/shared/form/CheckboxComponent';
+import { cleanup } from '@testing-library/react';
+
+describe('CheckboxComponent', () => {
```

Comment:
Missing tests for checking/unchecking it and checking if the event is triggered

Reply:
tests added